### ISSUE TITLE
[FR] - CLI to convert ITN pub/private keys to payment verification/signing keys

### ISSUE THREAD

**Issue by disassembler** [input-output-hk/cardano-node#1070]
**Describe the feature you'd like**
I need a cardano-cli command that can take an input of either a public or private ed25519 key and output the verification or signing key.

Example private key:

```
ed25519_sk1yhnetcmla9pskrvp5z5ff2v8gkenhmluy736jd6nrxrlxcgn70zsy94f7k
```

Corresponding public key:

```
ed25519_pk1demeytzdadayd4qrqeg2raadp2eceg3mrdmefxyfxx73q60hg4xsjjyzyq
```

**Describe alternatives you've considered**

**Additional context / screenshots**

**Comment by dcoutts** [comment]
Ok, so these are bech32 encoded (with prefix `ed25519_sk` or `ed25519_pk`) but are otherwise raw 32 byte ed25519 signing and verification keys.

**Comment by kevinhammond** [comment]
Is this related to the` to-verification` feature request?

**Comment by kevinhammond** [comment]
I believe this overlaps with #1065.  I will merge them.

Describe the feature you'd like
Implement to-verification key to make it easier for wrapper scripts to not have to specify a signing key as well as verification key (or address). This would also allow us to recreate the verification key using the signing key.

Similar feature is available for Byron CLI (and was also commonly used in Jormungandr)

**Comment by disassembler** [comment]
> I believe this overlaps with #1065. I will merge them.
> 
> Describe the feature you'd like
> Implement to-verification key to make it easier for wrapper scripts to not have to specify a signing key as well as verification key (or address). This would also allow us to recreate the verification key using the signing key.
> 
> Similar feature is available for Byron CLI (and was also commonly used in Jormungandr)

This doesn't overlap. The other issue is about taking an existing haskell generated skey and getting it's vkey from it. This issue is about converting either a ITN secret or public key and converting it to the equivalent haskell skey or vkey.

**Comment by Jimbo4350** [comment]
Implemented in #1136
